storage/acl/addBucketACL.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package acl
+
+// [START bucket_add_acl]
+import (
+	"context"
+
+	"cloud.google.com/go/storage"
+)
+
+// addBucketACL adds ACL to the specified bucket.
+func addBucketACL(bucket string) error {
+	// bucket := "bucket-name"
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return fmt.Errorf("storage.NewClient: %v", err)
```

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors